prow/lib.sh

```diff
@@ -33,6 +33,11 @@ function setup_and_export_git_sha() {
     elif [[ "${CI:-}" == 'prow' ]]; then
       # Set artifact dir based on checkout
       export ARTIFACTS_DIR="${ARTIFACTS_DIR:-${ARTIFACTS}}"
+      TMP_DIR=$PWD
+      cd ../ || return
+      # Clone the CNI repo so the CNI artifacts can be built.
+      git clone -b master "https://github.com/istio/cni.git"
+      cd "${TMP_DIR}" || return
```

Comment:
This is called by all e2e tests that run in prow. why is setting up cni needed here?

Reply:
@hklai - without this prow/release-test.sh will fail.   prow/release-test.sh runs the cloud_builder script but cloud_builder depends on preceeding steps to make sure the environment is setup correctly.   This is prow/lib.sh

Reply:
Can this be moved to prow/release-test.sh itself?

Or if needed be, a separate make target for cni?

Reply:
I should be able to reorganize in along those lines.